src/core_modules/capture-core/components/Pages/EnrollmentEditEvent/EnrollmentEditEventPage.container.js

```diff
@@ -1,31 +1,62 @@
 // @flow
 import React from 'react';
 // $FlowFixMe
-import { useSelector, shallowEqual } from 'react-redux';
+import { useSelector, shallowEqual, useDispatch } from 'react-redux';
+import { useHistory } from 'react-router';
 import { useProgramInfo } from '../../../hooks/useProgramInfo';
 import { pageMode } from './EnrollmentEditEventPage.const';
 import { EnrollmentEditEventPageComponent } from './EnrollmentEditEventPage.component';
+import { useWidgetDataFromStore } from '../EnrollmentAddEvent/hooks';
+import { useHideWidgetByRuleLocations } from '../Enrollment/EnrollmentPageDefault/hooks';
+import { urlArguments } from '../../../utils/url';
+import { deleteEnrollment } from '../Enrollment/EnrollmentPage.actions';
 
 export const EnrollmentEditEventPage = () => {
-    const { programId, stageId } = useSelector(
+    const history = useHistory();
+    const dispatch = useDispatch();
+    const { programId, stageId, teiId, enrollmentId, orgUnitId } = useSelector(
         ({
             router: {
                 location: { query },
             },
         }) => ({
             programId: query.programId,
             stageId: query.stageId,
+            teiId: query.teiId,
+            orgUnitId: query.orgUnitId,
+            enrollmentId: query.enrollmentId,
         }),
         shallowEqual,
     );
     const { program } = useProgramInfo(programId);
-    const showEditEvent = useSelector(({ viewEventPage }) => viewEventPage?.eventDetailsSection?.showEditEvent);
-    const programStage = [...program.stages?.values()].find(item => item.id === stageId);
+    const showEditEvent = useSelector(
+        ({ viewEventPage }) =>
+            viewEventPage?.eventDetailsSection?.showEditEvent,
+    );
+    const programStage = [...program.stages?.values()].find(
+        item => item.id === stageId,
+    );
+    const mode = showEditEvent ? pageMode.EDIT : pageMode.VIEW;
```

Comment:
Hey, 

What do you think about renaming this variable to `currentPageMode`? This will help to maintain homogeneity with the variable inside [WidgetEventEdit](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/WidgetEventEdit/WidgetEventEdit.container.js#L47)
Thank you!